### PR TITLE
Fix pipeline

### PR DESCRIPTION
**Pull request by ARPspoofing**

### CODE REVIEW

**Comment by ARPspoofing** [comment]
@JThh kindly review. 

I have commented out the comparison of typicalPersonAddressBook test since our json keeps on changing as it is not finalised.

I will uncomment it towards the end!

File: src/main/java/seedu/address/model/Model.java
Comment (JThh) [inline src/main/java/seedu/address/model/Model.java:1]: Can we possibly merge the methods for labs and tutorials and let them handle events in general?

File: src/main/java/seedu/address/model/event/UniqueLabList.java
Comment (JThh) [inline src/main/java/seedu/address/model/event/UniqueLabList.java:1]: Not sure if we can have a unified event list here!

**Review by JThh — APPROVED** [review]
LGTM! Just leave my 2 cents, and no worries if they cannot be fixed for now. Good to merge if CI is passed

File: src/main/java/seedu/address/model/event/UniqueLabList.java
Comment (ARPspoofing) [reply src/main/java/seedu/address/model/event/UniqueLabList.java:1]: sure I will change that in the future

File: src/main/java/seedu/address/model/Model.java
Comment (ARPspoofing) [reply src/main/java/seedu/address/model/Model.java:1]: Will change that in the future but in case we need different things can leave it first

**Pull request merged**